src/SimpleGraphs/generators/randgraphs.jl

```diff
@@ -281,11 +289,12 @@ julia> watts_strogatz(Int8(10), 4, 0.8, is_directed=true, seed=123)
 
 ### References
 - Collective dynamics of ‘small-world’ networks, Duncan J. Watts, Steven H. Strogatz. [https://doi.org/10.1038/30918](https://doi.org/10.1038/30918)
+- Scaling and percolation in the small-world network model, M. E. J. Newman, Duncan J. Watts. [https://doi.org/10.1103/PhysRevE.60.7332](https://doi.org/10.1103/PhysRevE.60.7332)
 - Small Worlds, Duncan J. watts. [https://en.wikipedia.org/wiki/Special:BookSources?isbn=978-0691005416](https://en.wikipedia.org/wiki/Special:BookSources?isbn=978-0691005416)
 """
 function watts_strogatz(
     n::Integer, k::Integer, β::Real;
-    is_directed=false, rng::Union{Nothing, AbstractRNG}=nothing, seed::Union{Nothing, Integer}=nothing
+    newman_watts=false, is_directed=false, rng::Union{Nothing, AbstractRNG}=nothing, seed::Union{Nothing, Integer}=nothing
```

Comment:
```suggestion
    newman_watts::Bool=false, is_directed=false, rng::Union{Nothing, AbstractRNG}=nothing, seed::Union{Nothing, Integer}=nothing
```
I would add a type here, although we also don't do it for other kwargs here. But that code is ancient.